channels_jsonrpc/jsonrpcconsumer.py

```diff
@@ -253,12 +267,12 @@ def __handle(self, content, message):
             else:
                 if isinstance(data, dict):
 
-                    if data.get('method') is not None and data.get('params') is not None and data.get('id') is None:
```

Comment:
You are right here, I did not think about such case. I'll change this back now...